src/components/db.js

```diff
@@ -0,0 +1,41 @@
+export const Workouts = [
+  {
+    id: 1,
+    day: 'day1',
+    parts: 'chest',
```

Comment:
Minor comment, because maybe you are going to add more than one part in the future!? Otherwise:

```suggestion
    part: 'chest',
```